### DISCUSSION TITLE
revert icons on ci.86box.net website?

### DISCUSSION THREAD

**Discussion by honx4amiga** [86Box/86Box#1785] [General]
is it possible to revert the icons on ci.86box.net website to the old weather-icons (sun, sun/cloud, cloud)?
i don't like the new icons. and the new design is causing a display issue, build numbers have now line break.
see screenshot. now i have to reduce display size to 90% in browser to see build number in one line again.
i didn't have that problem befor the new design on ci.86box.net website.

and wasn't there a build #3254 yesterday? it seems to be gone, now there there is #3253 latest build again.

![build#linebreak](https://user-images.githubusercontent.com/82286022/140537974-fa4caddb-33df-4993-98fb-9e0ca6470d4e.PNG)

**Comment by maxdlpee** [comment]
That build disappeared as a result of the Jenkins migration. It'll be recompiled once that's complete.

**Reply by honx4amiga** [reply]
is there no solution for this display problem?

there were no issues before that new "design" about a week ago.

**Reply by OBattler** [reply]
We upgraded Jenkins itself during the migration, and that came with the new design. I'm not even sure the old design can be used with the new Jenkins version and downgrading Jenkins back to an older version would expose us to security risks.
